docs/issue-management.md

```diff
@@ -0,0 +1,395 @@
+
+Size labels are specified according to lines of code; however, some issues may
+not relate to lines of code submission such as documentation. In those cases,
+use the labels to apply an equivalent complexity or size to the task at hand.
+
+Size labels include:
+
+* `size/XS` -- denotes a extra small issue, or PR that changes 0-9 lines, ignoring generated files
+* `size/S` -- denotes a small issue, or PR that changes 10-29 lines, ignoring generated files
+* `size/M` -- denotes a medium issue, or PR that changes 30-99 lines, ignoring generated files
+* `size/L` -- denotes a large issue, or PR that changes 100-499 lines, ignoring generated files
+* `size/XL` -- denotes a very large issue, or PR that changes 500+ lines, ignoring generated files
+
+Size labels are defined in [`docs/github-labels.md`](docs/github-labels.md).
+
+### Triage
+
+As soon as new issue are submitted, they must be triaged until they are ready to
+work. The maintainers may apply the following labels during the issue triage
+process:
+
+* `triage/duplicate`  -- indicates an issue is a duplicate of other open issue
+* `triage/needs-information` -- indicates an issue needs more information in order to work on it
+* `triage/not-reproducible` -- indicates an issue can not be reproduced as described
+* `triage/unresolved` -- indicates an issue that can not or will not be resolved
+
+Triage labels are defined in [`docs/github-labels.md`](docs/github-labels.md).
+
+### Lifecycle
+
+To track the state of an issue, the following labels will be assigned.
+
+* `lifecycle/active` -- indicates that an issue or PR is actively being worked on by a contributor
+* `lifecycle/frozen` -- indicates that an issue or PR should not be auto-closed due to staleness
+* `lifecycle/rotten` -- denotes an issue or PR that has aged beyond stale and will be auto-closed
+* `lifecycle/stale` -- denotes an issue or PR has remained open with no activity and has become stale
+
+The following schedule will be used to determine an issue's lifecycle:
+
+* after 90 days of inactivity, an issue will be automatically marked as `lifecycle/stale`
+* after 180 days of inactivity, an issue will be automatically marked as
+  `lifecycle/rotten` and can then be scheduled for auto closure
+* any issue marked as `lifecycle/frozen` will prevent automatic transitions to
+  stale or rotten state.
+* an issue marked as `lifecycle/active` adds an additional 30 days to the
+  scheduled stale and rotten transitions above. If no activity is present, the
+  issue can be automatically closed.
+
+Issue lifecycle management ensures that the project backlog remains fresh and
+relevent. Project maintainers and contributors will need to revisit issues to
+periodically assess their relevance and progress.
+
+TODO: Additional CI automation (GitHub actions) will be used to automatically
+apply and manage some of these lifecycle labels.
+
+Lifecycle labels are defined in [`docs/github-labels.md`](docs/github-labels.md).
```

Comment:
link not working because of missing leading slash

Reply:
Should have relative link so this work correctly in site docs. Have corrected.